src/lib/rpcWallet.js

```diff
@@ -126,23 +118,42 @@ export default {
     return extractHostName(url);
   },
   async shouldOpenPopup(aepp, action) {
```

Comment:
The most part of this function is checkConfirmations, I'm proposing to inline it.